src/Standards/Squiz/Sniffs/Commenting/PostStatementCommentSniff.php

```diff
@@ -109,6 +109,14 @@ public function process(File $phpcsFile, $stackPtr)
             }
         }
 
+        if ($phpcsFile->tokenizerType === 'PHP'
+            && preg_match('|//\s?@[^\s]+|', $tokens[$stackPtr]['content']) === 1
```

Comment:
This regex currently allows only for no or exactly one whitespace char. What is this based on ? Why would multiple spaces not be allowed ?

Also: should the `\s` (which includes horizontal space and new lines) be changed to the more specific `[ \t]` (only allow spaces or tabs) ?
Note: also means a test should be added with a tab between the `//` and the annotation.

Reply:
> This regex currently allows only for no or exactly one whitespace char. What is this based on ? Why would multiple spaces not be allowed ?

This was based on the PHPUnit behavior. It only recognizes `@codeCoverageIgnore` as an annotation if there is no or one space between `//` and the annotation. But after seeing your questions, I checked PHPStan and saw that it behaves differently. It accepts multiple spaces or tabs between `//` and the annotation (https://phpstan.org/r/5c00a06b-796b-4983-baca-64e93891f3ee). So, I went ahead and changed the regex to consider all variations supported by both PHPUnit and PHPStan.

> Also: should the \s (which includes horizontal space and new lines) be changed to the more specific [ \t] (only allow spaces or tabs) ?
Note: also means a test should be added with a tab between the // and the annotation.

Good point, I failed to consider this. I just updated the regex and added a test with a tab.